### DISCUSSION TITLE
Longest dimension

### DISCUSSION THREAD

**Discussion by jmauzyk** [dvdoug/BoxPacker#259] [Q&A] [answered]
Is it critical for the longest dimension of a box or item to be designated as its length or width? I know that some shipping tools (e.g. USPS) state that a box's longest dimension should be set as its length, not width. Does this need to be taken into account with BoxPacker, or is this a non-issue since BoxPacker attempts different rotations?

**Comment by dvdoug** [comment, marked as answer]
Hi @jmauzyk 

With reference to the diagram on https://boxpacker.io/en/stable/getting-started.html

What BoxPacker calls depth (the vertical dimension) is important as this is used for packing stability purposes (placing heavier/larger items at the bottom of the box as well as orientation of any items marked as keep-flat). The other 2 dimensions are interchangeable from BoxPacker's POV, width and length are just used as labels.

Hope that helps,
Doug
